cypress/e2e/tests/pages/explorer2/namespace-picker.spec.ts

```diff
@@ -191,40 +195,63 @@ describe('Namespace picker', { testIsolation: 'off' }, () => {
-
-  //   // get user id
-  //   cy.getRancherResource('v3', 'users?me=true').then((resp: Cypress.Response<any>) => {
-  //     const userId = resp.body.data[0].id.trim();
-
-  //     // create project
-  //     cy.createProject(projName, 'local', userId).then((resp: Cypress.Response<any>) => {
-  //       const projId = resp.body.id.trim();
+  it('newly created project/namespace appears in namespace picker', { tags: ['@explorer2', '@adminUser'] }, () => {
+    // get user id
+    cy.getRancherResource('v3', 'users?me=true').then((resp: Cypress.Response<any>) => {
+      const userId = resp.body.data[0].id.trim();
 
-  //       // create ns
-  //       cy.createNamespaceInProject(nsName, projId);
+      // create project
+      cy.createProject(projName, 'local', userId).then((resp: Cypress.Response<any>) => {
+        projectId = resp.body.id.trim();
 
-  //       // check ns picker
-  //       namespacePicker.toggle();
-  //       cy.contains(projName).should('be.visible');
-  //       cy.contains(nsName).should('be.visible');
+        // create ns
+        cy.createNamespaceInProject(nsName, projectId);
 
-  //       // delete project and ns
-  //       cy.deleteRancherResource('v1', 'namespaces', nsName);
-  //       cy.deleteRancherResource('v3', 'projects', projId);
+        // check ns picker
+        namespacePicker.toggle();
+        cy.contains(projName).should('exist').scrollIntoView().and('be.visible');
+        cy.contains(nsName).should('exist').scrollIntoView().and('be.visible');
+      });
+    });
+  });
 
-  //       // check ns picker
-  //       cy.reload();
-  //       namespacePicker.toggle();
-  //       cy.contains(projName, { timeout: 10000 }).should('not.exist');
-  //       cy.contains(nsName, { timeout: 10000 }).should('not.exist');
-  //     });
-  //   });
-  // });
+  it('deleted project/namespace gets removed from namespace picker', { tags: ['@explorer2', '@adminUser'] }, () => {
+    const projNameToDelete = `project-to-delete${ +new Date() }`;
+    const nsNameToDelete = `namespace-to-delete${ +new Date() }`;
+
+    // get user id
+    cy.getRancherResource('v3', 'users?me=true').then((resp: Cypress.Response<any>) => {
+      const userId = resp.body.data[0].id.trim();
+
+      // create project
+      cy.createProject(projNameToDelete, 'local', userId).then((resp: Cypress.Response<any>) => {
+        const projIdToDelete = resp.body.id.trim();
+
+        // create ns
+        cy.createNamespaceInProject(nsNameToDelete, projIdToDelete);
+
+        // check ns picker
+        namespacePicker.toggle();
+        cy.contains(projNameToDelete).should('exist').scrollIntoView().and('be.visible');
+        cy.contains(nsNameToDelete).should('exist').scrollIntoView().and('be.visible');
+
+        // delete project and ns
+        cy.deleteRancherResource('v1', 'namespaces', nsNameToDelete);
+        cy.deleteRancherResource('v3', 'projects', projIdToDelete);
+
+        // check ns picker
+        cy.reload();
```

Comment:
Is the reload necessary ?

Reply:
I just removed it and it worked without it locally